static/js/src/dynamic-forms.js

```diff
@@ -413,7 +413,7 @@ import { prepareInputFields } from "./prepare-form-inputs.js";
           const machinesInputs = howManyMachinesFieldset?.querySelectorAll(
             "input[name='how-many-machines-do-you-have']",
           );
-          machinesInputs.forEach((input) => {
+          machinesInputs?.forEach((input) => {
```

Comment:
I see what you mean, it does seem risky to remove all the name attributes like that. 

We could just remove the name attribute from the html itself - this is how I handle all fields that we don't want added to the payload

Reply:
There has to be a name attribute for radio inputs, otherwise it will behave more like a checkbox select rather than a radio select.
<img width="893" alt="Screenshot 2024-11-07 at 3 16 28 PM" src="https://github.com/user-attachments/assets/a701eb52-1b5e-4d12-a03a-fd801ec9e386">

Another idea I had was maybe to add a field on the form template to state that it is a radio field type and remove the name attributes from there. This way we don't have to hard-code removing specific name attributes and also control which radio fields to remove "name" from the form template. I don't have too much context on the form template implementation so it's just a rough suggestion. Thoughts? 🤓

Reply:
Ohh, I see, I did not know that.. When you say 'form template', do you mean the implementation that builds forms based on JSON data? As this isn't currently in use. If you are just referring to regular forms, adding a flag to radio groups we want to strip the name attribute from sounds like good to me. It would also be nice to add it to the generator, but we don't have to deal with that now.

Sorry for expanding the scope of this PR :see_no_evil:

Reply:
Yeah the form template refers to the JSON form generator. I'll proceed with adding a flag to radio groups, will apply those changes in the next commit! We can deal with the form generator later on, I mostly wanted to fix the modal issue since it's affecting all dynamic forms throughout the sites.

Reply:
Let me know when it is ready for review so we can get this merged ASAP